### ISSUE TITLE
Standby non-functional within Koreader (Kindle PW 4)

### ISSUE THREAD

**Issue by Rick684** [koreader/koreader#4377]
* KOReader version: KOReader 2018.12
* Device: Kindle Paperwhite 4 (2018) running 5.10.0.2 factory image.

#### Issue
I am unable to put my Kindle Paperwhite 4 (2018) in standby using the hardware button. The menu option for sleep also doesn't work. The only thing that happens is that my screen dims and a message comes on screen with an info(I) logo. 
In short, putting my Kindle in standby within Koreader doesn't work.

#### Steps to reproduce
Try to put the Kindle PW 4 in standby using either the hardware button or using the sleep button within Koreader. You will run into this issue.

##### `crash.log` (if applicable)
N/A (Koreader doesn't crash)

**Comment by Rick684** [comment]
> 
> 
> It could still be applicable; it'd also say if the suspend script returned some error message. :-)

In that case I will look up the log. Will report back soon :-)

**Comment by NiLuJe** [comment]
How are you launching KOReader?

If you're using the no framework buttons, don't, that doesn't work on FW 5.x. I should actually just hide them on FW 5.x.

If you're not using a SO device, then that's precisely what is supposed to happen: we only use the system's suspend screen on SO devices, everywhere else, we use our own suspend/screensaver handling.

As a sidenote: stuff is generally expected to be a bit wonky on factory FW, so expect some potentially weird unsupported and hard to reproduce issues.

**Comment by Rick684** [comment]
> 
> 
> How are you launching KOReader?
> 
> If you're using the no framework buttons, don't, that doesn't work on FW 5.x. I should actually just hide them on FW 5.x.
> 
> If you're not using a SO device, then that's precisely what is supposed to happen: we only use the system's suspend screen on SO devices, everywhere else, we use our own suspend/screensaver handling.
> 
> As a sidenote: stuff is generally expected to be a bit wonky on factory FW, so expect some potentially weird unsupported and hard to reproduce issues.

I am launching Koreader using KUAL. Never launched Koreader with the no framework option, I thought that would break things.
What does SO device stand for? Sorry, I'm not familiar with that acronym.

**Comment by Frenzie** [comment]
Special offers. It means it shows ads on standby, possibly at other times as well.

**Comment by Rick684** [comment]
Oh, I feel dumb. No, my Kindle is definitely without Special offers.

**Comment by NiLuJe** [comment]
I still can't tell what you actually think is a bug here ;).

**Comment by Rick684** [comment]
The bug is that my device doesn't go into the normal standby mode. Koreader only dims the screen and gives an [i Slaapstand] message.
As far as I understood your last message, this should only occur if my Kindle had SO enabled. And I am sure mine has not.

**Comment by Frenzie** [comment]
No, the standby message means your specific book doesn't have a cover image. You can pick a different behavior in Settings (gear icon) → Screensaver.

With special offers you should see an ad instead.

How can you tell that it's not going into "normal standby mode"?

**Comment by NiLuJe** [comment]
Yup, AFAICT, everything's going as expected here.
